src/core/algorithms/ind/spider/attribute.cpp

```diff
@@ -0,0 +1,38 @@
+/** \file
+ * \brief Spider attribute
+ *
+ * Attribute class methods definition
+ */
+#include "attribute.h"
+
+namespace algos::spider {
+
+std::vector<AttributeIndex> AINDAttribute::GetRefIds(config::ErrorType max_error) const {
```

Comment:
```suggestion
std::vector<AttributeIndex> AINDAttribute::GetRefIds(config::ErrorType const max_error) const {
```

Reply:
I believe that making double a constant parameter in the declaration is optional and not necessary. It is clear that in theory this can help avoid bugs when we accidentally change a variable that we did not want to change. But then it is necessary to change it everywhere, but I don’t see much point in this.

Reply:
Yes, indeed, a const can be added in a function definition without adding a const in the declaration. But I’m rather saying that if we want to add a constant in one place, then we should also add it in similar places. Because the code must be consistent at least within one pull request. Ideally, the coding style should be consistent throughout the entire project. But I have not seen such a use of constants for parameters in function definitions in desbordante.